packages/trpc/server/routers/viewer/teams/inviteMember/utils.ts

```diff
@@ -285,7 +285,7 @@ export async function createNewUsersConnectToOrgIfExists({
             data: {
               teamId: parentId,
               userId: createdUser.id,
-              role: input.role as MembershipRole,
+              role: MembershipRole.MEMBER,
```

Comment:
So earlier if you invite someone to team as admin then we would also assign admin role of complete org too?

Reply:
Yes